recipes/go-dep/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set goname = "github.com/golang/dep" %}
+{% set version = "0.5.0" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/", "\\" if win else "/") %}
+
+package:
+  name: go-{{ name|lower }}
```

Comment:
Are we starting to prefix Go packages generally?

Reply:
I think it is a good idea, because we will end eventually conflicting with other packages. I think we might as well start early.

Reply:
although I am not sure it makes sense to prefix with "go-" projects like kubernetes, singularity and terraform.

Reply:
Yeah, we went through similar gyrations with Python. Not really trying to suggest how it should be solved with Go.

More mentioning this as it seems we are developing a templating framework here for Go packages and wanted to get a sense to what extent this is needed/will be used in the future. Probably worth mentioning your thoughts on prefixing with Go packages somewhere with higher visibility like the webpage repo to get feedback from others.